src/ui.data-table/UTableRow.vue

```diff
@@ -40,38 +40,16 @@
         />
       </div>
 
-      <div
-        v-if="isCellObject(value)"
-        :class="cx([bodyCellPrimaryAttrs.class, getCellContentClasses(row, key)])"
-      >
-        <slot :name="`cell-${key}`" :value="value" :row="row">
-          <div v-bind="bodyCellPrimaryAttrs" ref="cellRef" :data-test="`${dataTest}-${key}-cell`">
-            {{ getCellPrimaryContent(value) }}
-          </div>
-
-          <div v-bind="bodyCellSecondaryAttrs">
-            <template v-if="Array.isArray(value.secondary)">
-              <div v-for="(secondary, idx) in value.secondary" ref="cellRef" :key="idx">
-                <span v-bind="bodyCellSecondaryEmptyAttrs">
-                  {{ secondary }}
-                </span>
-              </div>
-            </template>
-
-            <div v-else ref="cellRef">
-              {{ value.secondary }}
-            </div>
-          </div>
-        </slot>
-      </div>
-
-      <template v-else>
-        <slot :name="`cell-${key}`" :value="value" :row="row">
-          <div v-bind="bodyCellPrimaryAttrs" ref="cellRef" :data-test="`${dataTest}-${key}-cell`">
-            {{ value || value === 0 ? value : HYPHEN_SYMBOL }}
-          </div>
-        </slot>
-      </template>
+      <slot :name="`cell-${key}`" :value="value" :row="row">
```

Comment:
add `index` to slot prop, sometimes it may be useful.

And slot should be documented?

Reply:
It's documented in UTable component